### PR TITLE
added dictionary class and test cases for homoglyph

### PR DESCRIPTION
**Pull request by shashank-m**
Added dictionary.py in transforms folder which has a class
that returns a homoglyph for a particular character.

Added apply_homoglyph function in visual_sim_char class which does 
input : adversarial
output: 𝔞𝖉𝘃eⲅꜱ𝐚𝑟𑣃𝛼1

Added np.random.choice to unicode function(finally)

### CODE REVIEW

**Comment by Sharad24** [comment]
Is `dictionary.py` the most appropriate name?

**Comment by shashank-m** [comment]
@Sharad24 homodic.py? :). Also why are these builds failing. Not really familiar with unit testing and Travis.

File: decepticonlp/transforms/perturbations.py [hunk 1]
```diff
@@ -309,9 +310,43 @@ def apply(self, word: str, **kwargs):
 
         char_array = np.array(list(word))
 
-        int_pick = np.random.randint(0, high=unicode_array.shape[0], size=len(word))
+        int_pick = np.random.choice(unicode_array.shape[0], len(word))
 
         picked_unicode = unicode_array[int_pick]
```
Comment (Sharad24) [inline decepticonlp/transforms/perturbations.py:315]: This defeats the purpose of using np.random.choice. 

You can combine these two lines to use np.random.choice.

**Comment by Sharad24** [comment]
It would be ok to keep it `dictionary.py` as long as also add other used dictionaries used in here.

**Comment by Sharad24** [comment]
You can see what the errors are by clicking on details of whatever check is failing.

**Comment by shashank-m** [comment]
@Sharad24  it's an import error. i'm not able to import dictionary.py into perturbations.py . I tried relative import since they are in the same package but i get an error.

**Comment by shashank-m** [comment]
So should I create a new directory called dictioanry, initialise an empy init file in it and put the dictionary.py file? Like make the new directory in the transforms folder

**Comment by Sharad24** [comment]
Two errors. One for not reformatting your file and one for importing dictionary.py

**Comment by shashank-m** [comment]
@Sharad24 I made another PR after this where I just created a json file for the dictionary. I closed that but will put that up again. Even the json version was not working

**Pull request closed without merging**